tutorials/tutorial_e_signature_integration.md

```diff
@@ -0,0 +1,111 @@
+
+## Procore Tool Specific URL Schemes
+
+To further explain PDF file access and provide examples for various Procore tools, a general URL path scheme is shown below for each major tool that could benefit from e-signature integration.
+Each path assumes a base URL of https://app.procore.com.
+
+### Prime Contract
+
+- /{project_id}/project/prime_contracts/{prime_contract_id}.pdf
+
+### Commitments
+
+- /{project_id}/project/commitments/work_order_contracts/{work_order_contract_id}.pdf
+- /{project_id}/project/commitments/purchase_order_contracts/{purchase_order_id}.pdf
+
+### Owner Invoices
+
+- /{project_id}/project/prime_contracts/{prime_contract_id}/payment_applications/{payment_application_id}/pdf.pdf
+
+### Subcontractor Invoices
+
+- /{project_id}/project/commitments/work_order_contracts/{word_order_id}/requisitions/{requisition_id}.pdf
+
+### Change Orders
+
+- /{project_id}/project/commitments/purchase_order_contracts/{purchase_order_contract_id}/change_orders/commitment_contract_change_orders/{change_order_package_id}.pdf
+- /{project_id}/project/commitments/work_order_contracts/{work_order_contract_id}/change_orders/work_contract_change_orders/{change_order_package_id}.pdf
+- /{project_id}/project/prime_contracts/{prime_contract_id}/change_orders/change_order_packages/{change_order_package_id}.pdf
+
+### RFIs
+
+- (All repsonses) /{project_id}/project/rfi/view_pdf.pdf?id={rfi_id}&only_official=false
+- (Official responses only) /{project_id}/project/rfi/view_pdf.pdf?id={rfi_id}&only_official=true
+
+### Submittals
+
+- /{project_id}/project/submittal_logs/{submittal_id}.pdf
+
+### Daily Log
+
+- /{project_id}/project/daily_log/list.pdf?date={date}
+
+## Documents Tool Integration
+
+Additional information on integrating with the Documents tool can be found at [developers.procore.com](https://developers.procore.com/documentation/tutorial-uploads).
+We recommended that executed documents be saved to a folder within the Procore Documents tool for reference.
+This tool allows for Admin-level users to make folders private. We also recommend this approach for all contract documents.
+
+## Additional Functionality
+
+Additional functionality can be added to the integration to support contract execution.
+Through Procore’s API, an integration can update the status of a contract to “Complete,” mark the contract as executed and record the executed date.
+
+**Note:** Procore allows *Custom Workflows* to be assigned with these tools.
+Custom Workflow in this scenario mostly refers to internal “controls” or “approvals” prior to something being sent out.
+This might add some complexity to the update as it requires a different API call.
+(See information below.)
+
+Here are links to the API documentation covering the update actions discussed above for contracts, change orders, and workflows in Procore:
+
+### Contracts
+
+- [Prime Contract](https://developers.procore.com/reference/rest/v1/prime-contracts#update-prime-contract)
```

Comment:
Is this URL the correct one to publish in the guide? I opened it and it seems like it's marked as "alpha":

<img width="562" alt="Snap 2021-07-08 at 1 21 45 PM" src="https://user-images.githubusercontent.com/74197530/124985672-85ce4e00-dfef-11eb-944a-0ee14533c818.png">

Reply:
You are probably logged in as a Procore employee? The link is correct. External users will be taken to the v1.0 version of the resource which is in production.